### PR TITLE
created service for getting, posting and editing data

### PR DESCRIPTION
**Pull request by Chahlyk**

### CODE REVIEW

File: src/app/pages/record/record.service.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+  public addCategory(category: ICategories): Observable<ICategories> {
+    return this.http.post<ICategories>('categories', category);
+  }
+
+  public addEvent(event: IEvents): Observable<IEvents> {
+    return this.http.post<IEvents>('events', event);
+  }
+
+  public deleteCategory(id: number): Observable<unknown> {
```
Comment (Liliia-Abr) [inline src/app/pages/record/record.service.ts]: why unknown type? Why you think that you don't know type?

File: src/app/pages/record/record.service.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    return this.http.delete(`categories/${id}`);
+  }
+
+  public editCategory(id: number | undefined, category: ICategories): Observable<unknown> {
```
Comment (Liliia-Abr) [inline src/app/pages/record/record.service.ts]: how is id can be undefined? and why unknown?

Comment (Chahlyk) [reply src/app/pages/record/record.service.ts]: When I was creating the method in component.ts, I got an error in the console that the id is undefined, so I completed the type. What about unknow type, I forgot add ...put<ICategories>(... in function logic

File: src/app/pages/record/record.service.ts (hunk 1 above)
Comment (Chahlyk) [reply src/app/pages/record/record.service.ts]: Now I know and can correct this like in the previous comment

File: src/app/pages/record/record.service.ts (hunk 2 above)
Comment (Liliia-Abr) [reply src/app/pages/record/record.service.ts:29]: okay let's don't touch this and return to this questions later when we'll be working with this methods

**Review by Liliia-Abr — APPROVED** [review]

**Comment by Liliia-Abr** [comment]
okay let's don't touch this and return to this questions later when we'll be working with this methods

**Pull request merged**